time_tracker_api/projects/projects_model.py

```diff
@@ -63,6 +66,36 @@ def __init__(self):
             mapper=ProjectCosmosDBModel,
         )
 
+    def create_sql_in_condition(self, customer_ids_list):
+        id_values = convert_list_to_tuple_string(customer_ids_list)
+
+        return "c.customer_id IN {value_condition}".format(value_condition=id_values)
+    
+    def find_all_with_customer_id_in_list(
+        self,
+        event_context: EventContext,
+        customer_ids_list: List[str],
+        visible_only=True,
+        mapper: Callable = None,
+    ):  
+        visibility = self.create_sql_condition_for_visibility(visible_only)
+        query_str = """
+            SELECT * FROM c
+            WHERE {condition}
+            {visibility_condition}
+            """.format(
+            condition=self.create_sql_in_condition(customer_ids_list),
```

Comment:
We could call this function `create_sql_customer_id_in_condition` and use it with a variable for better readability.

Reply:
Or better talk with @kellycastrof, she thinks this may be a more generic function. You could try too.